public/assets/js/scripts.js

```diff
@@ -27,12 +27,22 @@ function initMap() {
 $(document).ready(function () {
   $('#payment, #profile, #security').hide();
   $(".search-ready-awesome-div").hide();
+  $('.search-calculate-button').prop('disabled', true);
 
   // initialize popovers
   $(function () {
     $('[data-toggle="popover"]').popover()
   });
 
+  let periodicChanged = false;
+  let initialChanged = false;
+    
+  function enableSeachCalculateBtn(periodicChanged, initialChanged) {
+    if (periodicChanged && initialChanged === true) {
```

Comment:
This can be simplified as
```suggestion
    if (periodicChanged && initialChanged) {
```